README.md

```diff
@@ -140,13 +140,25 @@ In order to use it, you need to create the blacklist table in a migration:
 def change
   create_table :jwt_blacklist do |t|
     t.string :jti, null: false
+    t.datetime :revoked_at, default: Time.now
+    t.datetime :expiration_time
   end
   add_index :jwt_blacklist, :jti
 end
 ```
-
 For performance reasons, it is better if the `jti` column is an index.
 
+Note, if you used the blacklist strategy in a previous version you may not have the fields *revoked_at* and *expiration_time.* If not run the following migration:
+```
+class AddRevokedAtAndExpirationTimeToJWTBlacklist < ActiveRecord::Migration
+  def change
+    add_column :JWTblacklist, :revoked_at, :datetime, default: Time.now
+    add_column :JWTblacklist, :expiration_time, :datetime
```

Comment:
Yes... in fact the table name is completely free. In the README I recommend `jtw_blacklist` because it is supposed to go in the scope of a larger application, while in the test I use `blacklist` because the fixture app is just about `jwt`. But feel free to change it if you like.